### DISCUSSION TITLE
Have some problem when installing weblate by Docker

### DISCUSSION THREAD

**Discussion by AlphaStarguide** [WeblateOrg/weblate#9410] [Q&A] [answered]
Need some help here.
When I run `docker compose up`, here are the error codes and it just loop the `cant remove` part.

weblate-docker-weblate-1   | 390 static files copied to '/app/cache/static'.
weblate-docker-weblate-1   | Compressing... done
weblate-docker-weblate-1   | Compressed 6 block(s) from 244 template(s) for 2 context(s).
weblate-docker-weblate-1   | rm: cannot remove '/var/log/nginx/access.log': Read-only file system
weblate-docker-weblate-1   | rm: cannot remove '/var/log/nginx/error.log': Read-only file system
weblate-docker-weblate-1 exited with code 1

Ubuntu 22, using the latest doc's install guide.
Dont know how to fix this.

**Comment by nijel** [comment, marked as answer]
Use `edge` tag instead of `latest`. There are chnages ongoing in the docker image.

**Reply by nijel** [reply]
The changes were released last week, the `latest` tag now works fine.
